CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing to TODO-Name
```

Comment:
Need to `s/TODO-Name/jsonml.js` (is that the name? `package.json` has `@condenast/jsonml.js`)

Reply:
whoops! missed that one!